app/views/recipes/_form.html.erb

```diff
@@ -26,9 +26,8 @@
     <div class="mt-3">
```

Comment:
don't change files that doesn't concern your task

Reply:
Thanks for the feedback, we'll put more attention!